### ISSUE TITLE
RuntimeError when using dataset cifar

### ISSUE THREAD

**Issue by thupchnsky** [oskopek/mvae#7]
**Describe the bug**
The following RuntimeError appears when I am trying to run run.py on cifar dataset.

**To Reproduce**
1. Set up your environment: `conda activate pt`
2. Run experiment with `python -m mt.examples.run --dataset="cifar" --model="e2,h2,s2" --fixed_curvature=False --epochs=200`
3. See error: Running on: cuda
VAE Model: e2,h2,s2; Epochs: 200; Time: 2021-01-26T16:19:41.074145; Fixed curvature: False; Dataset: cifar
TrainEpoch 0:   Traceback (most recent call last):
  File "/home/chaopan2/miniconda3/envs/pt/lib/python3.7/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/chaopan2/miniconda3/envs/pt/lib/python3.7/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/home/chaopan2/simulations/Py/p3/mvae/mt/examples/run.py", line 169, in <module>
    main()
  File "/home/chaopan2/simulations/Py/p3/mvae/mt/examples/run.py", line 162, in main
    max_epochs=args.epochs)
  File "/home/chaopan2/simulations/Py/p3/mvae/mt/mvae/models/train.py", line 118, in train_stopping
    train_results[self.epoch] = self._train_epoch(optimizer, train_data, beta=beta)
  File "/home/chaopan2/simulations/Py/p3/mvae/mt/mvae/models/train.py", line 188, in _train_epoch
    stats, (reparametrized, _, _) = self.model.train_step(optimizer, x_mb, beta=beta)
  File "/home/chaopan2/simulations/Py/p3/mvae/mt/mvae/models/vae.py", line 154, in train_step
    reparametrized, concat_z, x_mb_ = self(x_mb)
  File "/home/chaopan2/miniconda3/envs/pt/lib/python3.7/site-packages/torch/nn/modules/module.py", line 493, in __call__
    result = self.forward(*input, **kwargs)
  File "/home/chaopan2/simulations/Py/p3/mvae/mt/mvae/models/vae.py", line 74, in forward
    q_z, p_z, _ = component(x_encoded)
  File "/home/chaopan2/miniconda3/envs/pt/lib/python3.7/site-packages/torch/nn/modules/module.py", line 493, in __call__
    result = self.forward(*input, **kwargs)
  File "/home/chaopan2/simulations/Py/p3/mvae/mt/mvae/components/component.py", line 33, in forward
    z_params = self.encode(x)
  File "/home/chaopan2/simulations/Py/p3/mvae/mt/mvae/components/component.py", line 64, in encode
    z_mean = self.fc_mean(x)
  File "/home/chaopan2/miniconda3/envs/pt/lib/python3.7/site-packages/torch/nn/modules/module.py", line 493, in __call__
    result = self.forward(*input, **kwargs)
  File "/home/chaopan2/miniconda3/envs/pt/lib/python3.7/site-packages/torch/nn/modules/linear.py", line 92, in forward
    return F.linear(input, self.weight, self.bias)
  File "/home/chaopan2/miniconda3/envs/pt/lib/python3.7/site-packages/torch/nn/functional.py", line 1406, in linear
    ret = torch.addmm(bias, input, weight.t())
RuntimeError: size mismatch, m1: [100 x 8192], m2: [400 x 2] at /opt/conda/conda-bld/pytorch_1556653215914/work/aten/src/THC/generic/THCTensorMathBlas.cu:268

**Desktop (please complete the following information):**
 - OS: CentOS 7.7.1908
 - PyTorch version 1.1.0
 - Python version 3.7.3
 - other additional Python packages installed by requirements

**Comment by oskopek** [comment]
Sorry for the delay, will take a look at this this evening.

**Comment by oskopek** [comment]
This should be fixed in 28654a91079b95162cbe8139c3c8f9f0ec9f921e. Please let me know if it still doesn't work. To use the ConvVAE with CIFAR, you'll have to additionally specify `--h_dim`:

```
python -m mt.examples.run --dataset="cifar" --model="e2,h2,s2" --fixed_curvature=False --epochs=200 --architecture="conv" --h_dim=8192
```
